docs/code-guide.md

```diff
@@ -0,0 +1,941 @@
+﻿<header class="masthead">
```

Comment:
Unless this ends up targetting the web and needs it, the HTML should probably be removed. I think it's all fine with plain Markdown syntax